packages/@react-aria/utils/src/useLabels.ts

```diff
@@ -14,14 +14,19 @@ import {AriaLabelingProps, DOMProps} from '@react-types/shared';
 import {HTMLAttributes} from 'react';
 import {useId} from './useId';
 
+/**
+ * Merges aria-label and aria-labelledby into aria-labelledby when both exist.
```

Comment:
I wrote this before realizing this was out of scope, should I leave it in?

Reply:
Seems fine, doesn't quite cover everything that happens but fine for now.